docs/manual/existing_performance_counters.qbk

```diff
@@ -126,6 +126,18 @@ system and application performance.
         [Returns the overall execution time of all AGAS services provided by the
          given AGAS service category since its creation (in nanoseconds).]
     ]
+    [   [`/agas/count/entries`]
+        [`locality#*/total`
+
+          where:[br] `*` is the locality id of the locality the AGAS cache should
+          be queried. The locality id is a (zero based) number identifying the
+          locality.
+        ]
+        [None]
+        [Returns the number of cache entries resident in the AGAS cache of the
+        specified locality (see
+         `<cache_statistics>`.]
+    ]
```

Comment:
Thanks! Do we still have the NUMA domain specific counters? If yes, then it should be reflected in the docs. If not, the we can remove the NUMA-counter discoverer functions.